src/components/App.js

```diff
@@ -1,16 +1,23 @@
+/* eslint-disable no-unused-vars */
 import './App.css';
+import { useState } from 'react';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
+import isValidNumber from './logic/validNum';
 
```

Comment:
I think you need to import `Calculate.js` here not in `ButtonPanel.js`
You might need to adjust the codebase to follow the instruction in the pathwright, we both know that all students follow these instructions.
![image](https://user-images.githubusercontent.com/31897434/117814520-52c04600-b25c-11eb-9df2-0ee254445df9.png)
[Screenshot from pathwright](https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106376018/)